src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -378,4 +388,116 @@ public InputStream body() throws IOException {
         }
     }
 
+    /**
+     * Fake decorator.
+     * @since 0.16
+     */
+    final class Fake implements RqMultipart {
+
+        /**
+         * Fake multipart request.
+         */
+        private final RqMultipart fake;
+
+        /**
+         * Fake ctor.
+         * @param req Fake request header holder
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         * @todo #252:30min/DEV We should expect instances of Request as dispositions
```

Comment:
I don't understand what needs to be done. What do you want me to do, as a developer of this puzzle?